docs/sync.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Util script for synchronizing docs with frontpage repository.
+ * For running it:
+ * - cd /storybook/docs
+ * - node sync.js 
+ */
+const fs = require('fs');
+const readline = require('readline');
+const path = require('path');
+
+const askQuestion = (query) => {
+  const rl = readline.createInterface({
+      input: process.stdin,
+      output: process.stdout,
+  });
+
+  return new Promise(resolve => rl.question(`${query}\n`, ans => {
+      rl.close();
+      resolve(ans);
+  }))
+}
+
+const run = async () => {
+  let frontpageDocsPath = '/src/content/docs'
+
+  const frontpageAbsPath = await askQuestion('Provide the frontpage project absolute path:')
```

Comment:
I tested this with a relative path `../../frontpage` and it works fine. Suggest asking for a relative path, as the absolute path can be unwieldy and is seldom known off-hand.

Reply:
Perfect, I will update the command message for asking for a relative path and docs too.